index.js

```diff
@@ -2,4 +2,13 @@
 
 module.exports = {
   name: require('./package').name,
+
+  included(parent) {
+    this._super.included.apply(this, arguments)
+    console.warn(
+      `⚠️  DEPRECATION: [ember-metrics-pendo] Please use ember-metrics v1.4.0+ that includes pendo adapter and remove ember-metrics-pendo from your dependencies.
```

Comment:
As Pendo is not something general or generic I would add the article the before Pendo. However I suggest a slight difference:

```suggestion
      `⚠️  DEPRECATION: [ember-metrics-pendo] Please use ember-metrics v1.4.0+ which includes the adapter for Pendo and remove ember-metrics-pendo from your dependencies.
```